airflow/Dockerfile

```diff
@@ -46,7 +46,8 @@ RUN python3 -m venv --system-site-packages /stackable/app && \
     pip install --no-cache-dir --upgrade pip && \
     pip install --no-cache-dir apache-airflow[${AIRFLOW_EXTRAS}]==${PRODUCT} --constraint /tmp/constraints.txt && \
     # Needed for pandas S3 integration to e.g. write and read csv and parquet files to/from S3
-    pip install --no-cache-dir s3fs
```

Comment:
Can you check if this is really needed or if https://github.com/apache/airflow/pull/31835 helps?

Reply:
Their SBOM generation process is more complicated then ours, also their resulting SBOM lacks the dependency tree for Python packages. They do however contain NPM packages with their dependency relations, these are currently not included in the SBOM this PR generates. But Syft picks up the NPM components when scanning the image. We could look into generating an SBOM for the NPM components as well, the advantage would be the dependency tree information being present in our SBOMs. But this information is at the moment not really relevant for vulnerability analysis, so I think it's not worth the effort. We can always improve it later if we want to. For analysis of vulnerabilities the Python package dependency tree is useful, which is generated by this PR.